src/pages/explore.js

```diff
@@ -1,10 +1,96 @@
-import * as React from 'react';
-import Layout from '../components/Layout';
-
-const Explore = () => (
-  <Layout>
-    <h1>This is the explore page!</h1>
-  </Layout>
-);
+import { graphql, useStaticQuery } from "gatsby";
+import * as React from "react";
+import Layout from "../components/Layout";
+import ExploreCarousel from "../components/ExploreCarousel";
+
+const Explore = () => {
+  const exploreQuery = useStaticQuery(graphql`
+    query exploreQuery {
+      prismicExplorePage {
+        data {
+          sections {
+            section {
+              document {
+                ... on PrismicExploreSection {
+                  id
+                  uid
+                  data {
+                    title {
+                      text
+                    }
+                    works {
+                      work {
+                        document {
+                          ... on PrismicVideo {
+                            uid
+                            data {
+                              title {
+                                text
+                              }
+                              thumbnail {
+                                url
+                              }
+                              preview_text {
+                                richText
```

Comment:
even though this is technically a rich text field, since we're only using it for plain text you can use `text` here instead of `richText`, that way we don't need the extra work of using `<PrismicRichText>` 

same for PrismicArticle and PrismicGallery!